<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timetablePage</class>
 <widget class="QWidget" name="timetablePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель «График»</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCombineTimetable">
     <property name="text">
      <string>Объединять листы предварительной записи для выбранных суток</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSyncCheckableAndInvitiation">
     <property name="text">
      <string>Синхронизировать подтверждение с приглашением</string>
     </property>
     <property name="buddy">
      <cstring>chkSyncCheckableAndInvitiation</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="chkSyncCheckableAndInvitiation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>153</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAmbulanceUserCheckable">
     <property name="text">
      <string>Показывать подтверждение записей амбулаторного приема</string>
     </property>
     <property name="buddy">
      <cstring>chkAmbulanceUserCheckable</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkAmbulanceUserCheckable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>153</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDoubleClickQueuePerson">
     <property name="text">
      <string>Двойной щелчок в листе предварительной записи врача</string>
     </property>
     <property name="buddy">
      <cstring>cmbDoubleClickQueuePerson</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbDoubleClickQueuePerson">
     <item>
      <property name="text">
       <string>Изменить жалобы/примечания</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Перейти в картотеку</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Новое обращение</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="cmbCombineTimetable">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Нет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По назначению приёма</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Все</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbDoubleClickQueuePerson</tabstop>
  <tabstop>chkAmbulanceUserCheckable</tabstop>
  <tabstop>chkSyncCheckableAndInvitiation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
